.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,49 @@
+name: Pages
```

Comment:
Doesn't RTD take care of this?

Reply:
I don't have access to RTD's internals but most of the configuration should be set up through the YAML config. One exception is enabling PR builds.
Though, since it's hard to gain that access, it may be reasonable to set up a new one with several people added to prevent the bus factor problem.

I guess, it's fine to go through GH Pages for now. But it'd be nice to fix the RTD setup later.

One thing to do regardless is to run Sphinx in strict mode in CI for linting purposes. But this can also be done later.